test/test-multiple-mutators.c

```diff
@@ -0,0 +1,24 @@
+/**
+ * Test-Case for multiple custom mutators in C
+ * Reference:
+ * https://github.com/bruce30262/libprotobuf-mutator_fuzzing_learning/blob/master/4_libprotobuf_aflpp_custom_mutator/vuln.c
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+
+int main(int argc, char **argv) {
+
+  int  a = 0;
+  char s[16];
+  memset(s, 0, 16);
+  read(0, s, 0xa0);
```

Comment:
@rish9101 : this seems just to be a simple buffer overflow. Sorry, I can't see, how that tests _multiple_ mutators.

Reply:
the s buffer too small for the amount that is read, should be increased. and there should be two different mutators, and the 2nd sets [1] = 0 or something like that so we have a meaningful test.